### PR TITLE
[SD-4492]-(Imp) - Render timestamps in search and monitoring views us…

### PR DESCRIPTION
**Pull request by sjunaid**
…ing 24hr clock

### CODE REVIEW

File: scripts/superdesk/ui/ui.js [hunk 1]
```diff
@@ -1190,6 +1190,7 @@
 
         .config(['defaultConfigProvider', function(defaultConfig) {
             defaultConfig.set('ui.italicAbstract', true);
+            defaultConfig.set('shortTimeFormat', 'HH:mm'); // 24h format
```
Comment (petrjasek) [inline scripts/superdesk/ui/ui.js]: this setting should probably go into datetime module

File: scripts/superdesk/datetime/datetime_spec.js [hunk 2]
```diff
@@ -21,15 +22,18 @@
...
 
         describe('datetime service', function() {
-            it('can short format datetime', inject(function(datetime, moment) {
+            it('can short format datetime', inject(function(datetime, moment, config) {
                 var now = moment();
                 var day = now.isoWeekday();
 
-                expect(now.format('hh:mm')).toBe(datetime.shortFormat(now.format()));
+                var TIME_FORMAT = config.shortTimeFormat || 'hh:mm';
+                var WEEK_FORMAT = config.shortWeekFormat || 'dddd, ' + TIME_FORMAT;
```
Comment (petrjasek) [inline scripts/superdesk/datetime/datetime_spec.js]: imo it shouldn't use the config, it's testing the config, so you should set the value explicitly

**Comment by pavlovicnemanja** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**